src/algorithm/hnswlib/visited_list_pool.h

```diff
@@ -91,7 +87,7 @@ class VisitedListPool {
         VisitedList* rez;
         {
             std::unique_lock<std::mutex> lock(poolguard);
-            if (pool.size() > 0) {
+            if (!pool.empty()) {
```

Comment:
use `not` instead of `!` to make it clear

Reply:
done